.env

```diff
@@ -0,0 +1 @@
+DB_STRING=mongodb+srv://root:root@cluster1.x2wqw.mongodb.net/surveyApp?retryWrites=true&w=majority
```

Comment:
this is supposed to be .gitignore d (you should probably change your database's password now)